Tasks/ANT/task.json

```diff
@@ -11,9 +11,9 @@
     ],
     "author": "Microsoft Corporation",
     "version": {
-        "Major": 1,
+        "Major": 2,
```

Comment:
Are you breaking compat?  Why are you incrementing the major version.  Since you're adding capabilities but not breaking compat it should be a minor version bump